tests/Unit/Controllers/Lead/LeadSaveControllerTest.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Unit\Controllers\Lead;
+
+use App\Http\Controllers\Lead\LeadSaveController;
+use App\Models\Lead;
+use App\Models\User;
+use Illuminate\Http\Request;
+use Tests\TestCase;
+
+class LeadSaveControllerTest extends TestCase
+{
+    /** @test */
+    public function it_can_save_urls(): void
+    {
+        $user = User::factory()->create();
+        $this->actingAs($user);
+
+        $urls = [
```

Comment:
There is another way to do this with @dataProviders 

https://blog.martinhujer.cz/how-to-use-data-providers-in-phpunit/

Reply:
https://github.com/Roskus/prospect-flow-crm/pull/80/commits/1b0f7c63d67226331740305d2e10b6a208fc766f

[add Data Providers to Lead Save urls](https://github.com/Roskus/prospect-flow-crm/pull/80/commits/1b0f7c63d67226331740305d2e10b6a208fc766f)